### ISSUE TITLE
Pre-processing reference annotation method

### ISSUE THREAD

**Issue by kbergin** [HumanCellAtlas/secondary-analysis#240]
It was proposed that after #239 we should design a method to pre-process the annotation, removing the blacklisted genes, store the resulting artifact, and use it for pipeline runs. Should generalize across organisms (mouse human).

AC:
- [ ] Method pre-processes annotation
- [ ] Method is stored in skylab?
- [ ] Method has unit tests
- [ ] Artifact is stored in the proper place for production

Using the resulting artifact is in a separate ticket.
